### ISSUE TITLE
Feature: Stop words

### ISSUE THREAD

**Issue by gustavorps** [pandas-profiling/pandas-profiling#863]
**Missing functionality**
Able to pass a stop words list to improve Words tab details

**Proposed feature**
```py
profile.config.vars.cat.stop_words = [" ", "  ", "the", "and", "at"]
```

[`config.yaml`](https://github.com/pandas-profiling/pandas-profiling/blob/master/src/pandas_profiling/config_default.yaml)
```yaml
...
vars:
    ...
    cat:
        ...
        stop_words: [" ", "  ", "the", "and", "at"]
    ...
...
```

**Additional context**
![image](https://user-images.githubusercontent.com/516827/138155030-3dbdb283-2c0a-4790-8d4d-56b214ad9b3f.png)

**Comment by rluthi** [comment]
Hi, I'm happy take this one. This will be my first contribution

**Comment by rluthi** [comment]
This issue has been resolved in PR #877 which has been merged into the develop branch

**Comment by gustavorps** [comment]
Thanks @rluthi and @sbrugman
